envi/tests/test_arch_h8.py

```diff
@@ -0,0 +1,1084 @@
+        ( "1b53", 0x4560, 'dec.w #1, r3', IF_W, () ),
+        ( "1bf3", 0x4560, 'dec.l #2, er3', IF_L, () ),
+        ( "1f00", 0x4560, 'das r0h', 0, () ),
+        ( "5470", 0x4560, 'rts', IF_RET | IF_NOFALL, () ),
+        ( "4670", 0x4560, 'bne 45d2:8', IF_BRANCH | IF_COND, () ),
+        ( "4e90", 0x4560, 'bgt 44f2:8', IF_BRANCH | IF_COND, () ),
+        ( "58500070", 0x4560, 'bcs 45d4:16', IF_BRANCH | IF_COND, () ),
+        ( "58b0f070", 0x4560, 'bmi 35d4:16', IF_BRANCH | IF_COND, () ),
+        ( "01006df2", 0x4560, 'push.l er2', IF_L, () ),
+        ( "6dfa", 0x4560, 'push.w e2', IF_W, () ),
+        ( "6df2", 0x4560, 'push.w r2', IF_W, () ),
+        ( "6cda", 0x4560, 'mov.b r2l, @-er5', IF_B, () ),
+        ( '01006df2', 0x4560, 'push.l er2', IF_L, () ),
+        ( '6df2', 0x4560, 'push.w r2', IF_W, () ),
+        ( '6de2', 0x4560, 'mov.w r2, @-er6', IF_W, () ),
+        ( '6dda', 0x4560, 'mov.w e2, @-er5', IF_W, () ),
+        ( '01006dd2', 0x4560, 'mov.l er2, @-er5', IF_L, () ),
+        ( '6843', 0x4560, 'mov.b @er4, r3h', IF_B, () ),
+        ( '68c3', 0x4560, 'mov.b r3h, @er4', IF_B, () ),
+        ( '6ec34715', 0x4560, 'mov.b r3h, @(0x4715:16, er4)', IF_B, () ),
+        ( '6ecb4715', 0x4560, 'mov.b r3l, @(0x4715:16, er4)', IF_B, () ),
+        ( '01106d75', 0x4560, 'ldm.l @sp+, (er4-er5)', IF_L, () ),
+        ( '01106df4', 0x4560, 'stm.l (er4-er5), @-sp', IF_L, () ),
+
+
+        ( '8340', 0x0, 'add.b #40, r3h', IF_B, (
+            {'setup':(('r3h',0xca),('CCR_C',0)), 'tests':(('r3h',0x0a),('CCR_C',1),('CCR_V',0)) }, ) ),
+        ( '0832', 0x2, 'add.b r3h, r2h', IF_B, (
+            {'setup':(('r3h',0xca),('r2h',0x40), ('CCR_C',0)), 'tests':(('r2h',0x0a),('CCR_C',1),('CCR_V',0)) }, ) ),
+        ( '79134715', 0x4, 'add.w #4715, r3', IF_W, (
+            {'setup':(('r3',0xccca),('CCR_C',0)), 'tests':(('r3',0x13df),('CCR_C',1),('CCR_V',0)) }, ) ),
+        ( '0943', 0x8, 'add.w r4, r3', IF_W, (
+            {'setup':(('r3',0xccca),('r4',0x4715), ('CCR_C',0)), 'tests':(('r3',0x13df),('CCR_C',1),('CCR_V',0)) }, ) ),
+        ( '7A1300047145', 0xa, 'add.l #47145, er3', IF_L, () ),
+        ( '0AA3', 0x10, 'add.l er2, er3', IF_L, () ),
+        ( '0B03', 0x12, 'adds #1, er3', 0, () ),
+        ( '0B83', 0x14, 'adds #2, er3', 0, () ),
+        ( '0B93', 0x16, 'adds #4, er3', 0, () ),
+        ( '9340', 0x18, 'addx #40, r3h', 0, () ),
+        ( '0E43', 0x1a, 'addx r4h, r3h', 0, () ),
+        ( 'E340', 0x1c, 'and.b #40, r3h', IF_B, () ),
+        ( '1643', 0x1e, 'and.b r4h, r3h', IF_B, () ),
+        ( '79634715', 0x20, 'and.w #4715, r3', IF_W, () ),
+        ( '6643', 0x24, 'and.w r4, r3', IF_W, () ),
+        ( '7A6300047145', 0x26, 'and.l #47145, er3', IF_L, () ),
+        ( '01F06643', 0x2c, 'and.l er4, er3', IF_L, () ),
+        ( '0640', 0x30, 'andc #40, ccr', 0, () ),
+        ( '01410640', 0x32, 'andc #40, exr', 0, () ),
+        ( '7643', 0x36, 'band #4, r3h', 0, () ),
+        ( '7C307640', 0x38, 'band #4, @er3', 0, () ),
```

Comment:
curious about how you ended up with mixed case in your hex data declarations ( not a show stopper or anything, mostly just entertaining ;)